### PR TITLE
Add ftc commercial api client

### PR DESCRIPTION
**Pull request by beibei1989**

### CODE REVIEW

File: api_client/templates/fas.py [hunk 1]
```diff
@@ -410,7 +442,12 @@
     "path": "/api/v1/trial/",
     "method": "POST",
     "body": {
-        "sn": "{{ sn }}"
+        {% if sn is defined %}
+            "sn": "{{ sn }}"
+        {% endif %}
+        {% if customer_id is defined %}
```
Comment (jerryz1982) [inline api_client/templates/fas.py:448]: customer_id is not an accepted input.

File: api_client/templates/fascomm.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# Prefix         HTTP method
+# ADD_XXX      -->    POST
+# SET_XXX      -->    PUT
+# DELETE_XXX   -->    DELETE
+# GET_XXX      -->    GET
+# MODIFY_XXX   -->    PATCH
+
+# LOGIN
+
+import fas
```
Comment (jerryz1982) [inline api_client/templates/fascomm.py:28]: use full path

File: api_client/templates/fas.py [hunk 3]
```diff
@@ -348,6 +357,29 @@
 }
 """
 
+GET_AUTH = """
+{
+    {% if id is defined %}
+        "path": "/api/v1/auth/{{ id }}/",
+    {% else %}
+        {% set _options = {
+            "sn": sn,
```
Comment (jerryz1982) [inline api_client/templates/fas.py:366]: there is also a chunksize parameter.

**Review by jerryz1982 — COMMENTED** [review]
check in line

**Comment by samsu** [comment]
Thank you, the general api client added.

**Pull request closed without merging**